probe-rs/tests/source_location.rs

```diff
@@ -3,23 +3,23 @@ use std::path::{Path, PathBuf};
 
 const TEST_DATA: [(u64, u64, ColumnType); 8] = [
     // Target address, line, column
-    (0x920, 11, ColumnType::LeftEdge),
-    (0x922, 13, ColumnType::Column(21)),
-    (0x92e, 13, ColumnType::Column(11)),
-    (0x938, 27, ColumnType::Column(4)),
-    (0x93a, 14, ColumnType::Column(19)),
-    (0x956, 20, ColumnType::Column(12)),
-    (0x962, 21, ColumnType::Column(12)),
-    (0x96a, 22, ColumnType::Column(12)),
+    (0x80006EA, 220, ColumnType::Column(28)),
```

Comment:
Which source file does this relate to?

Line 220 in the test data repository does not have a column 28:

https://github.com/probe-rs/probe-rs-debugger-test/blob/7803a32ff16c0e74ccc73f67b953b41ea49df1ad/src/main.rs#L220

Reply:
Haha ... you're right. I built the binary just before I added support for the other chips. 

Now I'm having doubts if this strategy is going to work, because those tests will likely keep evolving.   

What do you suggest? Go and update the tests and then reference a specific commit in the test README?

Reply:
For me, the main goal of the tests is to avoid breaking existing functionality, so I hope that we don't have to change the tests that often. But I agree that the whole setup is a bit cumbersome right now.

Reply:
Changes made :-)

Is there some way we can use QEMU or similar to improve this?